lib/core/src/putUtil.cpp

```diff
@@ -677,17 +705,26 @@ int
 putDirUtil( rcComm_t **myConn, char *srcDir, char *targColl,
             rodsEnv *myRodsEnv, rodsArguments_t *rodsArgs, dataObjInp_t *dataObjOprInp,
             bulkOprInp_t *bulkOprInp, rodsRestart_t *rodsRestart,
-            bulkOprInfo_t *bulkOprInfo ) {
+            bulkOprInfo_t *bulkOprInfo )
+{
+    namespace fs = boost::filesystem;
+
     char srcChildPath[MAX_NAME_LEN], targChildPath[MAX_NAME_LEN];
+    int status = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'status' can be reduced.](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)